service/templates/projects.html

```diff
@@ -968,6 +968,54 @@ <h3>Periodic Builds</h3>
           </div>
         </div>
       </div>
+      <!-- Custom container resource aliases -->
+      <div class="variables">
+        <div class="form-group">
+          <div class="col-header col-lg-6 form-control-static">
+            <h3>Container Sizes</h3>
+            <div class="muted small">Specify custom computing resource configurations for running tasks in containers. Memory field denotes the memory (in MB) that the container will be allocated, and CPU is the CPU units that will be allocated. 1024 CPU units is equivalent to 1vCPU.</div>
+          </div>
+        </div>
+        <div id="container-sizes-list-header" class="form-group">
+          <div class="col-lg-2"> <label class="control-label"> Name </label> </div>
+          <div class="col-lg-2"> <label class="control-label"> Memory MB </label> </div>
+          <div class="col-lg-2"> <label class="control-label"> CPU </label> </div>
+          <div class="col-lg-2"></div>
+        </div>
+        <div id="container-sizes-list" class="form-group" ng-repeat="obj in settingsFormData.container_sizes track by $index">
+          <div class="col-lg-2">
+            <input class="form-control" ng-model="settingsFormData.container_size_names[$index]" type="text" placeholder="name">
```

Comment:
I'm not that good with JS, but since it seems like `container_size_names` is only needed to track the name here, is there no other way to just get the container size name from `ng-repeat` on  `container_sizes` keys? Can we do something like `ng-repeat="(name, obj) in settingsFormData.container_sizes`?

Reply:
That works! I tried something similar eariler but removed it because it was erroring out. This approach works though.